### DISCUSSION TITLE
Welcome to OpenEFT Discussions!

### DISCUSSION THREAD

**Discussion by NoahGWood** [NoahGWood/OpenEFT#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bongi11** [comment]
OpenEFT offer anyway to take scanned digital finger prints .nist file and convert to .EFT file for the alphabet boyz?
Noah neighbor im from Traverse City,mi

**Reply by Bongi11** [reply]
Nobody visit this anymore. Seems like it should be simple 🤷‍♂️

**Reply by trimfandango** [reply]
Hey Noah, I am not sure if you are still updating this. I used the local install a few times. First time I used JPG and it totally bugged out, stretching things, then I used a PNG that worked, but the eft was larger than the file limitation on eforms. finally I resized the image and ran again and got this error: "Fingerprint cards cannot be validated. Please retry or submit hard copy fingerprint cards. Additional error information: 1. Code: 1004
Data length too short. Length is 1 must between 3-3 in Field T2_HGT
I installed the NIST viewer to check what values were input but I couldn't tell what it was talking about.

**Reply by Hauburger** [reply]
@trimfandango I was messing with it yesterday there seems to be a duplication of the fingerprints which increases the size of the file.  Also i got the same error with a JPG and when i switched to PNG it fixed the errors.  I can send you the core.py and fingerprint.py files i modify to get it to work.  i got them to accept on the ATF site last night.
I do think he is modifying the code though to meet new requirements.  Just send me a message if you want the files i use and change them in the docker container.

**Reply by frostyjhammer** [reply]
I massaged the front end to more closely resemble the printed and hacked at the back-end enough to get it working.

Never submitted a Form, though, as I heard the game has changed. But I'm encouraged that @Hauburger got it working to the point where AFT accepted e-prints! 

@Hauburger did you push your code changes and start a Pull Request?
